packages/@aws-cdk/aws-s3/lib/bucket.ts

```diff
@@ -1840,7 +1840,9 @@ export class Bucket extends BucketBase {
     // objects in the bucket
     this.addToResourcePolicy(new iam.PolicyStatement({
       actions: [
-        ...perms.BUCKET_READ_ACTIONS, // list objects
+        // list objects
+        's3:GetBucket*',
```

Comment:
For consistency, put these in `perms.ts` under a new `BUCKET_READ_METADATA_ACTIONS` constant. 

Also - you have a linter violation here with indentation.